packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -101,6 +102,28 @@ export interface ViewableTree {
  */
 export interface ITree extends ViewableTree, IFluidLoadable {}
 
+/**
+ * {@link ITree} extended with some alpha APIs.
+ * @privateRemarks
+ * TODO: Promote this to alpha.
+ * @internal
+ */
+export interface ITreeAlpha extends ITree {
+	/**
+	 * Exports root in the same format as {@link TreeAlpha.(exportVerbose:1)} using stored keys.
+	 * @remarks
+	 * This is `undefined` iff the root field is empty (this can only happen if the root field is optional).
```

Comment:
Nit: I've been advocating for not using "iff", since I don't know how accessible that is to non-English-native speakers.
```suggestion
	 * This is `undefined` if and only if the root field is empty (this can only happen if the root field is optional).
```